packages/core/src/components/SimpleStepper/SimpleStepper.tsx

```diff
@@ -38,16 +44,22 @@ export const VerticalStepperContext = React.createContext<InternalState>({
 export interface StepperProps {
   elevated?: boolean;
   onStepChange?: (prevIndex: number, nextIndex: number) => void;
+  activeStep?: number;
 }
 
 export const SimpleStepper: FC<StepperProps> = ({
   children,
   elevated,
   onStepChange,
+  activeStep = 0,
 }) => {
-  const [stepIndex, setStepIndex] = useState<number>(0);
+  const [stepIndex, setStepIndex] = useState<number>(activeStep);
```

Comment:
wonder if you could use https://github.com/streamich/react-use/blob/master/docs/useCounter.md

Reply:
Not sure if it would bring any additional value though :) But theoretically I think it is possible